### PR TITLE
Update 'juju switch' to cover various usage shortcuts

### LINKED ISSUE #2537: Extend and enhance `switch` command
Since 2.0, it was possible to:

`juju switch foo` -> switches to the last used model in controller foo if it exists, otherwise model foo in the current controller
`juju switch :foo` -> switches to model foo in the current controller
`juju switch foo:bar` -> switches to model bar in controller foo

This is not well explained and so the current docs can be enhanced.

Secondarily, a recent PR introduced:

`juju switch foo:` -> switches to the last used model in controller foo

This is the extension to the docs (as per https://github.com/juju/juju/pull/7556).

### PR DESCRIPTION
**Pull request by pmatulis**
resolves #2537

### CODE REVIEW

File: src/en/models.md [hunk 1]
```diff
@@ -92,13 +92,30 @@ Common model management tasks are summarized below.
...
+   `juju switch [<controller or model>|<:model>|<controller>:<model>|<controller:>]`
    
    Running the command with no arguments will return the currently active 
    controller and model:
      
          juju switch
    
+   To change to a model:
+   
+   <!-- WARNING: EXTRA SPACES AT EOL -->
```
Comment (evilnick) [inline src/en/models.md]: we don't need this, we just need to shoot anyone who strips spaces

File: src/en/models.md [hunk 2]
```diff
@@ -92,13 +92,30 @@ Common model management tasks are summarized below.
...
+   
+   `juju switch foo`  
+   Selects the last used model in controller 'foo' (if the latter exists),
```
Comment (evilnick) [inline src/en/models.md]: does this mean if you have a controller foo, and a model bar:foo then:

juju switch bar
juju switch foo

will take you to back to the foo container?

Comment (pmatulis) [reply src/en/models.md]: If controller foo exists, the command `juju switch foo` will, irrespective of previous commands ("where you are"), always take you to the last used model in controller foo.

Comment (evilnick) [reply src/en/models.md]: how confusing

**Pull request merged**